tests/functional/test_interactive.py

```diff
@@ -39,3 +39,12 @@ def test_ipython(pyi_builder):
 def test_pyside(pyi_builder):
     pyi_builder.test_script('pyi_interact_pyside.py', #pyi_args=['--windowed'],
                             runtime=_RUNTIME)
+
+
+@xfail(reason="May fail on CI - Should succeed on local run")
```

Comment:
Why should this fail on CI? It's setup for headless GUI testing - the PyQt tests require it. We want CI to pass unless I'm missing something.

Reply:
The xfail is actually a relic of the previous PR. 
I had left it in for the time being, since I did not know if Tcl/Tk can successfully create a GUI on CI, since `test_libraries::test_tkinter` only tests the paths and does not create a GUI.
I wanted to decide after CI has run through for the first time whether the test remains marked with xfail or gets other options, since I don't know exactly whether the same headless GUI setup can also test Tcl/Tk.